### PR TITLE
 [ADD] new module pos_cash_control_multiple_config to handle correctly cash control in a multi point of sale context

### PR DESCRIPTION
**Pull request by legalsylvain**
Hi. 

Recently face a problem in point_of_sale module, if cash control is enabled, and in a multiple point of sale context  : the opening balance is not the correct one.

This module fixes the problem

Complete Bug report is available there :  https://github.com/odoo/odoo/issues/62147

Thanks : CC : @OCA/pos-maintainers, @alexis-via, @quentinDupont,

@grap, ref #/board/144/card/1103

### CODE REVIEW

**Review by quentinDupont — APPROVED** [review]

File: pos_cash_control_multiple_config/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Point Of Sale - Correct Opening Balance",
+    "summary": "Handle correctly opening balance in a multi point of sale"
+    " context with Cash control enabled.",
+    "version": "12.0.1.0.1",
+    "category": "Point of Sale",
+    "author": "GRAP, Odoo Community Association (OCA)",
+    "website": "http://www.grap.coop",
```
Comment (MeetKD) [inline pos_cash_control_multiple_config/__manifest__.py]: You can change website to https://github.com/OCA/pos/

File: pos_cash_control_multiple_config/models/account_bank_statement.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+class AccountBankStatement(models.Model):
+    _inherit = "account.bank.statement"
+
+    @api.multi
+    def _get_opening_balance(self, journal_id):
+        PosSession = self.env["pos.session"]
+        pos_config_id = self.env.context.get("pos_config_id")
+        if self.env.context.get("pos_config_id"):
```
Comment (MeetKD) [inline pos_cash_control_multiple_config/models/account_bank_statement.py]:
```suggestion
        if pos_config_id:
```

File: pos_cash_control_multiple_config/models/account_bank_statement.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+                # returning 0
+                return 0
+            else:
+                last_valid_statement = False
+                for old_statement in sessions.mapped('statement_ids'):
+                    if old_statement.journal_id.id == journal_id:
+                        last_valid_statement = old_statement
+                if last_valid_statement:
+                    return last_valid_statement.balance_end
+
```
Comment (MeetKD) [inline pos_cash_control_multiple_config/models/account_bank_statement.py:30]: We can also remove this extra line.

**Review by MeetKD — CHANGES_REQUESTED** [review]
Minor Improvements.

**Comment by legalsylvain** [comment]
Hi @MeetKD . Could you update your review ? 

thanks !

**Review by MeetKD — APPROVED** [review]
Looks good!

**Pull request merged**